cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from datetime import timedelta
+
+from fixtures.pytest_store import store
+from cfme.optimize.bottlenecks import Bottlenecks
+from utils import conf
+from utils.appliance.implementations.ui import navigate_to
+from utils.appliance import get_or_create_current_appliance
+from utils.ssh import SSHClient
+from utils.version import current_version
+
+db_tbl = store.current_appliance.db['bottleneck_events']
+db_events = store.current_appliance.db.session.query(db_tbl.timestamp,
+    db_tbl.resource_type, db_tbl.resource_name, db_tbl.event_type, db_tbl.severity, db_tbl.message)
+
+
+@pytest.fixture(scope="session")
+def extend_db_partition():
+    app = get_or_create_current_appliance()
```

Comment:
All these `extend`, `backup` and `restore` fixtures are inspired by `test_db_migrate` fixture here: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/test_db_migrate.py#L24

The DB partition is extended once per session and the backup & restore must work correctly. I believe we should potentially have a yield fixture for backup & restore available at the base level, further down the line, to be used from anywhere - which has to work 100%.